libs/ui-watt/src/lib/components/tooltip/watt-tooltip.directive.spec.ts

```diff
@@ -35,14 +35,50 @@ async function setup() {
 }
 
 describe(WattTooltipDirective.name, () => {
+  const getTooltip = () => screen.getByText('Click me');
+  const getTooltipTarget = () => screen.getByText('Button');
+
+  const isTooltipVisible = (): boolean =>
+    getTooltip().classList.contains('show');
+
   it('always displays accessible tooltip', async () => {
     await setup();
-    expect(screen.getByRole('tooltip', { hidden: true })).toBeInTheDocument();
+    expect(screen.getAllByRole('tooltip')[0]).toBeInTheDocument();
   });
```

Comment:
FYI, I think this test was meant to ensure that tooltips are always present in the DOM, to allow for things like screen readers to see them without requiring hover. I don't think that is the case with with this implementation, since the tooltips are not created until `show()` is triggered. No worries though, as long as we don't have any strict requirements regarding accessibility it probably doesn't matter.

Reply:
Actually this is not the case, the tooltips are still created initially. It's only `popper` which are initialized on show :)